graph-builder/src/graph.rs

```diff
@@ -55,6 +59,17 @@ lazy_static! {
         "Total number of upstream scrapes"
     )
     .unwrap();
+    static ref GRAPH_UPSTREAM_INITIAL_SCRAPE: Gauge = Gauge::new(
+        "graph_initial_upstream_scrape_duration",
+        "Duration of initial upstream scrape"
+    )
+    .unwrap();
+    static ref UPSTREAM_SCRAPES_DURATION: Histogram = Histogram::with_opts(histogram_opts!(
+        "graph_upstream_scrapes_duration",
+        "Upstream scrape duration in seconds",
+        UPSTREAM_SCRAPE_BUCKETS.to_vec()
```

Comment:
Same comment as in #154: I'd inline the value here. If you decide to keep it separate, then there's a small naming nit where we're using the plural "scrapes" on other occurrences.

Reply:
Fixed